drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/pages/ActionInsertFactFieldsPageViewImpl.ui.xml

```diff
@@ -63,36 +66,36 @@
         </gwt:VerticalPanel>
       </gwt:HorizontalPanel>
       <gwt:VerticalPanel ui:field="patternDefinition" visible="false">
-        <gwt:HorizontalPanel ui:field="bindingContainer" styleName="{res.css.wizardDTableFieldContainerValid} form-group">
-          <b:Label text="{i18n.Binding}:" styleName="{res.css.wizardDTableFields} help-block"/>
-          <guvnor:BindingTextBox ui:field="txtBinding" enabled="false" addStyleNames="{res.css.wizardDTableFields}"></guvnor:BindingTextBox>
+        <gwt:HorizontalPanel ui:field="bindingContainer" styleName="{res.css.wizardDTableFieldContainerValid}">
+          <b:Label text="{i18n.Binding}:" styleName="{res.css.wizardDTableFields}"/>
```

Comment:
Have a look if you could reuse org.uberfire.client.views.pfly.widgets.FormLabelHelp or org.uberfire.client.views.pfly.widgets.HelpIcon in here. Another point is that you probably want to use FormLabel instead of Label.